src/uniprotkb/config/UniProtKBColumnConfiguration.tsx

```diff
@@ -703,33 +703,67 @@ UniProtKBColumnConfiguration.set(UniProtKBColumn.ccInteraction, {
     const interactionComments = data[EntrySection.Interaction].commentsData.get(
       'INTERACTION'
     ) as InteractionComment[];
-    return (
-      <ExpandableList displayNumberOfHiddenItems>
-        {interactionComments?.map((interactionCC) =>
-          interactionCC.interactions.map((interaction) => (
-            <div
-              key={
-                interaction.type === InteractionType.SELF
-                  ? 'self'
-                  : `${interaction.interactantOne.uniProtKBAccession}-${interaction.interactantTwo.uniProtKBAccession}`
-              }
-            >
-              {interaction.type === InteractionType.SELF
-                ? 'Itself'
-                : interaction.interactantOne.uniProtKBAccession && (
-                    <Link
-                      to={getEntryPath(
-                        Namespace.uniprotkb,
-                        interaction.interactantOne.uniProtKBAccession
-                      )}
-                    >
-                      {interaction.interactantOne.uniProtKBAccession}
-                    </Link>
+    const { primaryAccession } = data;
+
+    const interactionElements = interactionComments
```

Comment:
This needs to return a unique set of links. I'd rather have it return a `Set` of accessions, or the specific string `'self'`, and then generate a link out of those. This should also remove the need for these complex keys in there, and the `div` look like they are not necessary